zebra/zebra_fpm.c

```diff
@@ -1993,6 +1996,9 @@ static int zfpm_init(struct thread_master *master)
 	zfpm_stats_init(&zfpm_g->last_ivl_stats);
 	zfpm_stats_init(&zfpm_g->cumulative_stats);
 
+	memset(&ipv4ll_gateway, 0, sizeof(ipv4ll_gateway));
+	inet_pton(AF_INET, ipv4_ll_buf, &ipv4ll_gateway.ipv4);
```

Comment:
so still confused about this - aren't these four octets completely static, completely fixed? and aren't they exactly the same as the four octets in another module? do we really need two copies of those four octets?

Reply:
As per comment from Donald, tried to keep the fpm code separate from other dataplane module while coping the same behaviour.  Also since the variable needed an init call to be set, was not sure about including it in a static lib.

Reply:
but ... the "variable" in this case is just a constant - the code is taking a constant string and turning it into four octets. the four octets are completely constant. there's no need for init apis or calls to inet_pton() - there's no need for the strings either, is there?

Reply:
Thanks for merging the fix. I think the intent of the original code in rt_netlink.c could have been to make the variable initialisation architecture/platform independent. We can explore how it can be done in platform independent way without using init call and  inet_pton().